editor/post-title/index.js

```diff
@@ -80,8 +83,10 @@ class PostTitle extends Component {
 		this.setState( { isSelected: false } );
 	}
 
-	handleClickOutside() {
-		this.setState( { isSelected: false } );
+	blurIfOutside( event ) {
+		if ( ! this.nodes.container.contains( event.relatedTarget ) ) {
```

Comment:
Shouldn't this mean that the `textarea` blurring if filtered out? Why are you needing to stop propagation in the clipboard container? Is the relatedTarget not set?

Reply:
It turns out it isn't. I thought `relatedTarget` was fragile. It may be that it is set in situations where something else has received a specific focus() call, and isn't set in situations where the focused element has been suddenly removed from the page and the body gets focus by default. In this situation, the active element seems to be body, but relatedTarget is nothing (when the textarea is removed).